tests/cli_test_case.py

```diff
@@ -48,6 +51,7 @@ def setUp(self):
 
     def tearDown(self):
         clean_session_files()
+        del os.environ[SESSION_DIR_KEY]
```

Comment:
With `tempfile.mkdtemp`, you need to clean up the temp file manually https://stackoverflow.com/questions/6884991/how-to-delete-a-directory-created-with-tempfile-mkdtemp/55871731

Reply:
I fixed it